ctapipe_io_magic/exceptions.py

```diff
@@ -0,0 +1,25 @@
+class MissingInputFilesError(Exception):
+    """
+    Exception raised when there are no input files.
+    """
+
+    def __init__(self, message):
```

Comment:
This __init__ function looks like the default one of of Exception class, so I do not think it is needed to write it here explicitely, you could make it simpler:
```
class MissingInputFilesError(Exception):
    """
    Exception raised when there are no input files.
    """
    pass
```

Reply:
Done.